psij-ci-setup

```diff
@@ -21,13 +21,23 @@ cron_check() {
     echo "$(($S1 || ! $S2))"
 }
 
+crontab_list() {
+    # wraps crontab -l such that an empty string is returned for a missing
+    # crontab rather than an error
+    if crontab -l 2>&1 | grep "no crontab for" >/dev/null 2>&1 ; then
+        echo ""
+    else
+        crontab -l
+    fi
+}
+
 
 cron_check_existing() {
-    crontab -l 2>/dev/null | grep "psij-ci-run" >/dev/null 2>&1
+    crontab_list 2>/dev/null | grep "psij-ci-run" >/dev/null 2>&1
```

Comment:
We should be able to remove the first `/dev/null` redirect now - but it doesn't hurt either...

Reply:
Good catch. I removed those.